src/lib/daccBikeGoal.js

```diff
@@ -0,0 +1,63 @@
+import { DACC_MEASURE_NAME_BIKE_GOAL } from 'src/constants'
+import { buildBikeCommuteTimeseriesQueryByAccountId } from 'src/queries/queries'
+
+import flag from 'cozy-flags'
+import log from 'cozy-logger'
+
+import {
+  createMeasureForDACC,
+  sendMeasureToDACCWithRemoteDoctype
+} from './dacc'
+import { countUniqDays } from './timeseries'
+
+/**
+ * The group name provided by flag
+ *
+ * @param {import("cozy-client/dist/index").CozyClient}} client - The cozy-client instance
+ * @returns {string} The group name
+ */
+export const getBikeGroupName = async client => {
+  await flag.initialize(client)
```

Comment:
note to myself: we should provide a `runService` command that handle the flag init for us.

Reply:
Where would be this command?